pallets/committee/src/lib.rs

```diff
@@ -431,6 +439,36 @@ pub mod pallet {
 			Self::deposit_event(Event::NewConstituent(constituent));
 			Ok(())
 		}
+
+		/// Remove council or constituent via governance
+		///
+		/// This call can only be called after the approval of the committee
+		#[pallet::weight((T::WeightInfo::remove_member(), DispatchClass::Operational))]
+		pub fn remove_member(origin: OriginFor<T>, member: AccountIdFor<T>) -> DispatchResult {
+			T::ApprovedByCommitteeOrigin::ensure_origin(origin)?;
+
+			Members::<T>::try_mutate_exists(member.clone(), |m| -> Result<(), DispatchError> {
```

Comment:
We can probably borrow here?
```suggestion
			Members::<T>::try_mutate_exists(&member, |maybe_member| -> Result<(), DispatchError> {
```

Reply:
We can beautify this a bit, so that 
* the closure `|maybe_member| -> Result<MemberType, DispatchError>` returns the member type of the removed member, so we fire the event outside of `try_mutate_exists `
* maybe use something like `maybe_member.as_mut().filter(|| is_council).map_err(..)?` to get rid of the verbose if let Some